website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -65,6 +65,43 @@ The most common types are:
  - `Gauge` Value can be increased and decreased as needed.
 5. `_total` suffix should be used for accumulating count. If your metrics has labels with high cardinality, like `pod`/`container` it usually means that you can aggregate it more, thus it will not require `_total` suffix.
 
+#### Recommended Metrics
+
+{{% alert title="Warning" color="warning" %}}
+The following operator health metric is new and may be subjected to change 
+{{% /alert %}}
+* `<operator-name-prefix>_operator_health_status` - The metric reports the operator health status. It should be reported by the operator itself based on the internal operator validations.  
```

Comment:
It seems like we should name this simply `operator_health_status` (no operator name prefix)? The operator name will be in the `name` label already so no need to duplicate it here, right?

Reply:
I investigated this and there is an issue in Prometheus if the `Help` text is reported differently with the same metric name. I believe having a separated metric name for each operator with the same suffix would be the better way here. Also the `name` label will allow for adding  the full and "pretty" name, but its only optional.

Reply:
See here https://prometheus.io/docs/instrumenting/writing_exporters/#help-strings
```
Prometheus doesn’t like one metric having different help strings. If you’re making one metric from many others, choose one of them to put in the help string.
```

Reply:
This brings up another concern: what happens when we add a fourth enumerated state? We'll have to change the help text to mention that state, we won't be able to ensure that every operator installed on a cluster will update at the same time to use that new help text.

This is another reason to go with a string label for the state enumeration. We can add self-describing state strings without changing the help text.

Reply:
I don't see a possibility for a fourth enumerated state.
There are 3 states: healthy / lost important functionality / about to loss important functionality(optional)
Having strings adds complications. I dont believe we should encourage developers to add self-describing state strings. As I said before, numbers are much more strict, which is important when you are proposing a metric that each operator needs to implement separatly.